app/steps/expenses_additional.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ExpensesAdditional < Step
+  step_attributes \
```

Comment:
super tiny stylistic thing: in other step classes we've been putting parens around step_attributes attributes rather than using `\` -- also, alpha order?